src/gallia/commands/scan/uds/sa_dump_seeds.py

```diff
@@ -84,6 +85,13 @@ def configure_parser(self) -> None:
             default=b"",
             help="Append an optional data record to each seed request",
         )
+        self.parser.add_argument(
+            "--sleep",
+            default=0,
```

Comment:
nitpick: Remove this, then you can do a `not None` check below. Would also work better for gallia 2.0 with stricter typing.

```suggestion
```